opm/simulators/wells/SegmentState.cpp

```diff
@@ -16,45 +16,59 @@
   You should have received a copy of the GNU General Public License
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
+
 #if HAVE_CONFIG_H
 #include "config.h"
 #endif // HAVE_CONFIG_H
 
-#include <algorithm>
-#include <stdexcept>
-
 #include <opm/simulators/wells/SegmentState.hpp>
 
-#include <opm/input/eclipse/Schedule/Well/WellConnections.hpp>
 #include <opm/input/eclipse/Schedule/MSW/WellSegments.hpp>
 
-namespace Opm
-{
+#include <algorithm>
+#include <cstddef>
+#include <iterator>
+#include <stdexcept>
+#include <vector>
 
 namespace {
-std::vector<int> make_segment_number( const WellSegments& segments ) {
-    std::vector<int> segment_number;
-    std::transform(segments.begin(), segments.end(), std::back_insert_iterator(segment_number), [](const Segment& segment) { return segment.segmentNumber(); });
-    return segment_number;
-}
 
+    std::vector<int> make_segment_number(const Opm::WellSegments& segments)
```

Comment:
indent for namespace?

Reply:
> indent for namespace?

Nice catch.  I've pushed an update to remove that.